templates/main_simulate.cpp

```diff
@@ -43,18 +70,26 @@ int main(int argc, char **argv) {
         std::string resultPath = argv[6];
         std::string simulationMode = argv[7];
 
+        // TODO: testing-only remove result file
+        remove(resultFileName.c_str());
+
+        auto dpPath = conditionFilePath;
+        {
+            // check if this is a result file or a new input file
+            // TODO: this should be handled cleaner
+            auto file = parpe::hdf5OpenForReading(conditionFileName);
+            if(parpe::hdf5GroupExists(
+                        file.getId(),
+                        (conditionFilePath + "/inputData").c_str()))
+                dpPath = conditionFilePath + "/inputData";
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Variable 'dpPath' is assigned a value that is never used.](https://app.codacy.com/manual/dweindl/parPE/pullRequest?prid=4904108)